### DISCUSSION TITLE
How can I switch to annother Github mirror?

### DISCUSSION THREAD

**Discussion by wizardforcel** [microsoft/vcpkg#33480] [Q&A]
For example, `https://gitclone.com/github.com/<org>/<repo>`

My current way is to start VSCODE on VCPKG root dir, and then replace all `github.com` into `gitclone.com/github.com`.

Is there any convenient ways?

**Comment by autoantwort** [comment]
Afaik no

**Comment by Neumann-A** [comment]
> Is there any convenient ways?

A convenient way no. But a hacky way. You can simply overwrite the function `vcpkg_from_github` from the triplet for example.
